Gopkg.toml

```diff
@@ -30,7 +30,8 @@ required = [
     "github.com/golang/protobuf/protoc-gen-go",
     "github.com/gogo/protobuf/protoc-gen-gogo",
     "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway",
-    "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger"
+    "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger",
+    "github.com/russross/blackfriday"
```

Comment:
why do we need it here?

Reply:
It's needed by cobra and the automatic transitive resolution didn't work.